### ISSUE TITLE
Can MegaDepth depth values output be converted to Kinect Depth?

### ISSUE THREAD

**Issue by encodingwaddles** [lixx2938/MegaDepth#15]
Is there a way we can convert MegaDepth depth values output into something similar to Kinect camera output? I'm guessing it's more than just scaling the values to the range of Kinect camera's (0,2048) ? 

Thanks in advance!

**Comment by krishvishal** [comment]
I don't think so because MegaDepth doesn't predict metric depth. That means it doesn't know the scale information. The predicted depth map is relative not absolute. Here is a quote from the paper. 

> Our method does not predict metric depth; future work in SfM could use
> learning or semantic information to correctly scale scenes.
